plugins/plugin.go

```diff
@@ -51,12 +52,13 @@ var version = "UNSET"
 // metrics will need
 func New(options ...Option) (*Plugin, error) {
 	newPlugin := &Plugin{
-		Name:            "default",
-		Role:            "",
-		PollingInterval: 10,
-		MetricsProto:    "http",
-		MetricsHost:     "localhost",
-		MetricsPort:     "61001",
+		Name:              "default",
+		Role:              "",
+		PollingInterval:   10,
+		MetricsProto:      "http",
+		MetricsHost:       "localhost",
+		MetricsPort:       "61001",
```

Comment:
you can easily convert int to string. Th reason why i'd love to see it as int, is because it is easier to validate. Next PR might be adding some validation login and converting string to int just for validation looks gross.